presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestAggregationFunction.java

```diff
@@ -38,19 +38,46 @@
 import static com.facebook.presto.operator.aggregation.AggregationTestUtils.assertAggregation;
 import static com.facebook.presto.sql.analyzer.TypeSignatureProvider.fromTypeSignatures;
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static java.util.Objects.requireNonNull;
 
 public abstract class AbstractTestAggregationFunction
 {
     protected TypeRegistry typeRegistry;
     protected FunctionManager functionManager;
     protected Session session;
+    protected FeaturesConfig featuresConfig;
+
+    protected AbstractTestAggregationFunction()
+    {
+        this(testSessionBuilder().build());
```

Comment:
You can probably just use `TEST_SESSION`.

Reply:
`TEST_SESSION` and `testSessionBuilder().build()` create different sessions, so I left the one that was previously in place, which was `testSessionBuilder().build()`